dawn.json

```diff
@@ -1028,15 +1036,23 @@
                 "args": [
                     {"name": "num mip levels", "type": "uint32_t"}
                 ]
-            },
-            {
-                "name": "set allowed usage",
-                "args": [
-                    {"name": "usage", "type": "texture usage bit"}
-                ]
             }
         ]
     },
+    "texture descriptor": {
+        "category": "structure",
+        "extensible": true,
+        "members": [
+            {"name": "usage", "type": "texture usage bit"},
+            {"name": "dimension", "type": "texture dimension"},
+            {"name": "width", "type": "uint32_t"},
+            {"name": "height", "type": "uint32_t"},
+            {"name": "depth", "type": "uint32_t"},
+            {"name": "arrayLayer", "type": "uint32_t"},
```

Comment:
`arrayLayer` LGTM, we might need to rename it when WebGPU decides what name it wants to use.

Reply:
Also FYI, there will be default values for these members so we don't have to type `arrayLayers = 1` every time. Likewise this will be a bit later when WebGPU crystallizes.